public/app/core/components/AppChrome/History/HistoryWrapper.tsx

```diff
@@ -78,127 +82,205 @@ function HistoryEntryAppView({ entry, isSelected, onClick }: ItemProps) {
+                      <Text color="secondary" variant="bodySmall">
+                        {view.description}
+                      </Text>
+                    )}
+                  </Stack>
+                </Card>
+              );
+            })}
+          </div>
+        )}
       </Stack>
-      {isExpanded && (
-        <div className={styles.expanded}>
-          {views.map((view, index) => {
-            return (
-              <Card
-                key={index}
-                href={view.url}
-                onClick={() => {
-                  store.setObject('CLICKING_HISTORY', true);
-                  onClick();
-                }}
-                isCompact={true}
-                className={view.time === selectedViewTime ? undefined : styles.card}
-              >
-                <Stack direction="column" gap={0}>
-                  <Text variant="bodySmall">{view.name}</Text>
-                  {view.description && (
-                    <Text color="secondary" variant="bodySmall">
-                      {view.description}
-                    </Text>
-                  )}
-                </Stack>
-              </Card>
-            );
-          })}
-        </div>
-      )}
-    </Stack>
+    </span>
   );
 }
 const getStyles = (theme: GrafanaTheme2) => {
   return {
     card: css({
+      label: 'card',
+      background: 'none',
+      margin: theme.spacing(0.5, 0),
+    }),
+    cardSelected: css({
+      label: 'card-selected',
+      background: 'none',
+    }),
+    subCard: css({
+      label: 'subcard',
       background: 'none',
+      margin: 0,
     }),
     iconButton: css({
+      label: 'expand-button',
       margin: 0,
     }),
+    iconButtonCircle: css({
+      label: 'blue-circle-icon',
+      margin: 0,
+      background: theme.colors.background.primary,
+      fill: theme.colors.primary.main,
+      cursor: 'default',
+      '&:hover:before': {
+        background: 'none',
+      },
+      //Need this to place the icon on the line, otherwise the line will appear on top of the icon
+      zIndex: 0,
+    }),
+    iconButtonDot: css({
+      label: 'blue-dot-icon',
+      margin: 0,
+      color: theme.colors.primary.main,
+      border: theme.shape.radius.circle,
+      cursor: 'default',
+      '&:hover:before': {
+        background: 'none',
+      },
+      //Need this to place the icon on the line, otherwise the line will appear on top of the icon
+      zIndex: 0,
+    }),
     expanded: css({
+      label: 'expanded',
       display: 'flex',
       flexDirection: 'column',
-      marginLeft: theme.spacing(5),
+      marginLeft: theme.spacing(6),
       gap: theme.spacing(1),
       position: 'relative',
       '&:before': {
         content: '""',
         position: 'absolute',
-        left: theme.spacing(-2),
+        left: 0,
         top: 0,
         height: '100%',
         width: '1px',
         background: theme.colors.border.weak,
       },
     }),
+    borderLeft: css({
```

Comment:
we should try and name the styles in a more semantic fashion. `timeline` maybe?

same with `paddingLeft`/`marginBottom` below